source/extensions/access_loggers/common/grpc_access_logger.h

```diff
@@ -135,10 +136,42 @@ template <typename LogRequest, typename LogResponse> class GrpcAccessLogClient {
     return true;
   }
 
+  Http::AsyncClient::StreamOptions createStreamOptionsForRetry() {
+    auto opt = Http::AsyncClient::StreamOptions();
+
+    if (!grpc_stream_retry_policy_) {
+      return opt;
+    }
+
+    // TODO(shikugawa): refactor conversion from route retry policy to core retry policy.
+    // This conversion is duplicated with jwks_fetcher.cc.
```

Comment:
Lets do this in another PR before this lands.

Reply:
done https://github.com/envoyproxy/envoy/pull/17803